packages/gen-ai-hub/src/client/openai/openai-types.ts

```diff
@@ -100,7 +100,7 @@ export interface OpenAiChatAssistantMessage {
   /**
    * Message content.
    */
-  content: string | null;
+  content?: string;
```

Comment:
I thought this was a generated type and could be null as well, e.g. if for some reason the content from the API is actually `null`?